README.rst

```diff
@@ -41,7 +41,11 @@ Some noteworthy flags::
 
     codespell -w, --write-changes
 
-The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.::
+The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
+
+When ignoring false positives, note that allowed words are *case-sensitive*, but dictionary matches are *case-insensitive*. For example, the dictionary entry ``wrod`` will match ``Wrod``, but to allow it you must pass ``wrod``.
```

Comment:
How about this tweak? Do you think this helps, or doesn't hinder the clarity?
```suggestion
When ignoring false positives, note that spelling errors are *case-insensitive* but words to ignore are *case-sensitive*. For example, the dictionary entry ``wrod`` will also match the typo ``Wrod``, but to ignore it you must pass ``wrod``.
```

Reply:
This is much better!